processor/src/chiplets/memory/segment.rs

```diff
@@ -167,9 +167,9 @@ impl MemorySegmentTrace {
         let (word_addr, addr_idx_in_word) = addr_to_word_addr_and_idx(addr);
 
         match self.0.entry(word_addr) {
+            // If this is the first access to the ctx/word pair, then all values in the word
+            // are initialized to 0, except for the address being written.
```

Comment:
typo: should it say "... except for when the address is being written to."?

Reply:
Yes, fixed